src/sconsx/tools/qhull.py

```diff
@@ -78,22 +78,36 @@ def option( self, opts):
 
          ('qhull_libs_suffix',
            'Qhull library suffix name like -vc80 or -mgw',
-           self._default['libs_suffix'])
+           self._default['libs_suffix']),
+
+         BoolVariable('WITH_QHULL',
+           'Specify whether you want to compile your project with QHULL', True)
      )
 
 
    def update(self, env):
       """ Update the environment with specific flags """
 
-      env.AppendUnique(CPPPATH=[env['qhull_includes']])
-      env.AppendUnique(LIBPATH=[env['qhull_lib']])
+      if env['WITH_QHULL'] :
+        def_qhull_inc = env['qhull_includes']
+        qhull_inc = pj(def_qhull_inc, 'libqhull')
+        if not os.path.exists(os.path.join(def_qhull_inc, "qhull_a.h")) and not os.path.exists(os.path.join(qhull_inc, "qhull_a.h")) :

```

Comment:
Cut this line in two lines to ease. We just need to look at qhull_a.h, isn't it?

Reply:
Indeed we look at qhull_a.h but 2 possible directory of header exists and are tested.

Reply:
It would be clearer to use path.py for the path management. But I will do that on another PR